src/test/java/no/nav/fo/veilarbregistrering/sykemelding/adapter/SykmeldtInfoClientTest.java

```diff
@@ -66,18 +66,22 @@ public void setup() {
         }; //Noop, vi trenger ikke kafka
         AktiveringTilstandRepository aktiveringTilstandRepository = mock(AktiveringTilstandRepository.class);
 
+        OppfolgingGatewayImpl oppfolgingGateway = new OppfolgingGatewayImpl(oppfolgingClient);
+
         brukerRegistreringService =
                 new BrukerRegistreringService(
                         brukerRegistreringRepository,
                         profileringRepository,
-                        new OppfolgingGatewayImpl(oppfolgingClient),
+                        oppfolgingGateway,
                         personGateway,
-                        new SykemeldingService(new SykemeldingGatewayImpl(sykeforloepMetadataClient)),
                         arbeidsforholdGateway,
                         profileringService,
                         arbeidssokerRegistrertProducer,
                         arbeidssokerProfileringProducer,
-                        aktiveringTilstandRepository);
+                        aktiveringTilstandRepository,
+                        new HentBrukerTilstandService(
+                                oppfolgingGateway,
+                                new SykemeldingService(new SykemeldingGatewayImpl(sykeforloepMetadataClient))));
```

Comment:
Dette er ikke helt bra, men en walkaround / utsettelse av gamle testdata. Denne testen tester alt for mye ift. eget scope, og krever litt omskriving. Den er tung å forstå slik den fremstår nå.